src/desktop/scene/canvasview.cpp

```diff
@@ -441,14 +442,14 @@ void CanvasView::focusInEvent(QFocusEvent *event)
 	m_keysDown.clear();
 }
 
-canvas::Point CanvasView::mapToScene(const QPoint &point, qreal pressure, qreal xtilt, qreal ytilt, qreal rotation) const
+canvas::Point CanvasView::mapToScene(long long timeMsec, const QPoint &point, qreal pressure, qreal xtilt, qreal ytilt, qreal rotation) const
```

Comment:
Should this time input be using the event `timestamp`?

Reply:
No, I had tried that, but it didn't work. Event timing is used for the stabilizer. It eats timestamps from input events and periodic polling. `QTimerEvent` doesn't have a `timestamp` field, so we have to use epoch milliseconds instead. Input event timestamps aren't compatible with that, for me those are in milliseconds since X was started for example. And I couldn't find an avenue to convert between those, so I punted to using epoch time for both. If there's a way to get the current input timestamp or to covert an event timestamp to epoch milliseconds, then we could use the event `timestamp` instead though.